docs/UserGuide.md

```diff
@@ -54,75 +56,87 @@ open the help window.<br>
 
 * Commands are usually abbreviated to facilitate faster typing for user convenience. As a general rule of thumb, `m`, `e` and `t` are used to represent `Member`, `Event` and `Task` respectively.
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+* Words in `UPPER_CASE` are the parameters to be supplied by the user, and they are preceded with prefixes of format `/prefix `<br>
   e.g. in `madd /n NAME`, `NAME` is a parameter which can be used as `madd /n John Doe`.
 
 * Items in square brackets are optional.<br>
   e.g. `/n NAME [/p POSITION]` can be used as `/n John Doe /p friend` or as `/n John Doe`.
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[/p POSITION]…​` can be used as ` ` (i.e. 0 times), `/p friend`, `/p friend /p family` etc.
+  e.g. `[/p POSITION]…​` can be completely absent (i.e. 0 times), `/p friend`, `/p friend /p family` etc.
+
+* All prefixes that are provided must be followed by a space and a non-empty parameter, except `/p ` prefix in `medit`, `/m ` prefix in `eedit`, `/att` and `/abs` in `mlist`, and `/ovd` in `tlist`.
```

Comment:
Maybe can be written as 

* All prefixes that are provided must be followed by a space and a non-empty parameter.
There are exceptions to this rule :
1. XXXX
2. XXXX
3. XXXX